src/twisted/internet/base.py

```diff
@@ -719,7 +719,9 @@ def fireSystemEvent(self, eventType):
             event.fireEvent()
 
 
-    def addSystemEventTrigger(self, _phase, _eventType, _f, *args, **kw):
+    # We need to ignore mypy errors here because the '_f' parameter is
+    # not in IReactorCore.addSystemEventTrigger
+    def addSystemEventTrigger(self, _phase, _eventType, _f, *args, **kw):  # type: ignore[override]  # noqa
```

Comment:
I think that this should follow the signature from the interface: `def addSystemEventTrigger(phase, eventType, callable, *args, **kw):` and rename `_phase` -> `phase` and `_f` -> `callable`

Reply:
I've moved this to: https://github.com/twisted/twisted/pull/1325